unic/char/range/src/step.rs

```diff
@@ -0,0 +1,40 @@
+use std::char;
```

Comment:
If the problem is `std::char` and the primitive type `char` conflicting, that's an intended overlap. The `char` module contains things like `char::from_u32` and `char::MAX` which should act like they are on the primitive `char`.

If you're saying `unic-char`, that's not referenced anywhere from my code. The only time there is a `char` module there is from inside the main `unic` crate.